domain/snyk/codeaction.go

```diff
@@ -43,15 +43,15 @@ type CodeAction struct {
 	// DeferredEdit is a function that returns a WorkspaceEdit.
 	// Used for heavy calculations that shouldn't be done ahead of time.
 	// A CodeAction cannot have both Edit and DeferredEdit.
-	DeferredEdit *func() *WorkspaceEdit
+	DeferredEdit *func() *WorkspaceEdit `json:"-"`
```

Comment:
Why this?

Reply:
func() serialization was failing when marshalling. In case of persisting scan results I think this can be ignored